README.md

```diff
@@ -38,25 +38,27 @@ This is the Javascript implementation and it works both in **Browsers** and **No
 ## Table of Contents
 
 <!-- toc -->
+- [Table of Contents](#table-of-contents)
 - [Install](#install)
   - [Using NodeJS](#using-nodejs)
   - [In a web browser](#in-a-web-browser)
-	- [In a web browser through Browserify](#through-browserify)
-	- [In a web browser through Webpack](#through-webpack)
-	- [In a web browser through CDN](#from-cdn)
+    - [**through Browserify**](#through-browserify)
```

Comment:
@deanpress Did you made these list elements bold on purpose? or it's your prettier config?

Reply:
Anyways, I don't think it's a roadblock. I'll fix the bold list items myself :)

Reply:
@vasa-develop Yeah that was my auto-formatter. My bad :D